eclipse-scout-core/src/desktop/outline/pages/Page.ts

```diff
@@ -433,6 +435,11 @@ export class Page extends TreeNode implements PageModel {
     }
     this.detailForm = form;
     if (form) {
+      form.one('destroy', () => {
```

Comment:
Maybe remove the listener when the form changes, see FormMenu or FormTableControl